www/static/js/docs.js

```diff
@@ -40,7 +40,7 @@ $(document).ready(function () {
 
     // Table of Contents
     $('#page-toc').toc({
-        'selectors':         'h1,h2,h3,h4,h5,h6', // elements to use as headings
+        'selectors':         'h1,h2', // elements to use as headings
```

Comment:
Confusing for who? I don't think users expect the TOC to be that comprehensive; major headings only should be fine. We do need to be vigilant when we write pages to make sure the TOC displays correctly. I would argue that defining/using other tags just complicates the markdown and defeats the purpose of using it.